src/autocomplete.js

```diff
@@ -21,7 +21,7 @@ const hasPointerEvents = (() => {
 })()
 
 function isIosDevice () {
-  return !!(navigator.userAgent.match(/(iPod|iPhone|iPad)/g) && navigator.userAgent.match(/AppleWebKit/g))
```

Comment:
The `!!` here coerces the value to a boolean, if you don't do it then you'll return the value of the `.match` function which is a string, and could be confusing.

```js
> isIosDevice()
'AppleWebKit'
```

vs

```js
> isIosDevice()
true
```

Reply:
Thanks @tvararu ! Will remove this change :)